StreamChatClient_v3/Workers/Background/ChannelEventsHandler_Tests.swift

```diff
@@ -63,6 +63,6 @@ class WebSocketClientMock: WebSocketClient {
 
 class APIClientMock: APIClient {
     init() {
-        super.init(apiKey: "", baseURL: URL(string: "test")!, sessionConfiguration: .default)
+        super.init(apiKey: APIKey("test_app")!, baseURL: URL(string: "test")!, sessionConfiguration: .default)
```

Comment:
`"test_app` is a static string, seems like compiler is not picking the correct init

Reply:
Interesting

Reply:
fixed in 6793e3301810e0d7658d51aa01ac85db5b35ac87